Monika After Story/game/script-farewells.rpy

```diff
@@ -1543,3 +1543,51 @@ label bye_prompt_workout:
     $ persistent._mas_greeting_type_timeout = datetime.timedelta(hours=4)
     $ persistent._mas_greeting_type = store.mas_greetings.TYPE_WORKOUT
     return 'quit'
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_prompt_hangout",
+            prompt="I'm going to hang out with friends.",
+            unlocked=True,
+            pool=True
+        ),
+        code="BYE"
+    )
+
+label bye_prompt_hangout:
+    if mas_isMoniNormal(higher=True):
+        if mas_getEVL_shown_count("bye_prompt_hangout") == 0:
+            if persistent._mas_pm_has_friends:
+                m 1eua "Alright, [player]."
+                m 3hub "I know, you should introduce me to them sometime!"
+                m 6eub "If they're your friends, I'm sure I'd like them!"
+            else:
+                m 3eub "I'm glad you're finding friends to go out with [player]!"
+                m 1rud "I know I'd like to spend as much time with you as possible..."
+                m 1hua "But I'd also like you to have friends you can meet face to face!"
+        else:
+            if persistent._mas_pm_has_friends:
+                m 1eua "Alright, [player]."
+                m 3hub "Have you told them about us yet?"
+                m 1hub "Ahaha!"
+            else:
+                m 1wua "Again? That's exciting!"
+                m 3eua "I hope you've made a really good friend lately."
+                m 3hua "Maybe tell them about me sometime? I hope they're nice."
+                m 1rusdla "What am I saying? Of course they're nice, they're your friend after all."
+
+        m 1eub "Anyway, see you later~"
+
+    if mas_isMoniDis():
```

Comment:
.
```suggestion
    elif mas_isMoniDis(higher=True):
```

The `higher=True` parameter is assuming Upset will share this path, as right now there is no upset path. This dialogue fits upset, so I think that works. Either way, this should be an elif here